docs/sources/explore/logs-integration.md

```diff
@@ -97,6 +97,17 @@ You can change the order of received logs from the default descending order (new
 
 Each log row has an extendable area with its labels and detected fields, for more robust interaction. For all labels we have added the ability to filter for (positive filter) and filter out (negative filter) selected labels. Each field or label also has a stats icon to display ad-hoc statistics in relation to all displayed logs.
 
+### Escaping newlines
+
+Explore automatically detects some incorrectly escaped sequences in log lines, such as newlines (`\n`, `\r`) or tabs (`\t`). When it detects such sequences, Explore provides an "Escape newlines" option.
+
+To automatically fix detected incorrectly escaped sequences:
+
+1. Click "Escape newlines" to replace the sequences.
+2. Manually review the replacements to confirm their correctness.
+
+After replacing these sequences, Explore changes the option from "Escape newlines" to "Remove escaping". Clicking this option reverts the replacements.
```

Comment:
```suggestion
If you want to revert the replacements, then click the "Remove escaping" option.
```

Reply:
Because this involves a UI element changing, and not two separate UI elements, the "Remove escaping" option won't always be visible. This suggestion would mean the conditions for that UI element's presence won't be documented. Is this intended?

Reply:
I've added an edit to see if we can make that more clear.